projects/appimagekit.cmake

```diff
@@ -1,5 +1,7 @@
 if(BUILD_OS_LINUX)
     ExternalProject_Add(AppImageKit
+        #GIT_REPOSITORY https://github.com/probonopd/AppImageKit.git
```

Comment:
And if we want to jump into the latest code of probonopd/AppImageKit?
I just added it here to be able to switch between the latest work in probonopd/AppImageKit for testing and the current release.